doc/source/io.rst

```diff
@@ -1949,56 +1949,102 @@ module and use the same parsing code as the above to convert tabular data into
 a DataFrame. See the :ref:`cookbook<cookbook.excel>` for some
 advanced strategies
 
-Besides ``read_excel`` you can also read Excel files using the ``ExcelFile``
-class. The following two commands are equivalent:
+Reading Excel Files
+~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 0.16
+``read_excel`` can read more than one sheet, by setting ``sheetname`` to either
```

Comment:
You need to add a blank line between  `.. versionadded:: 0.16` and the explanation (otherwise sphinx gives warnings)